content/javascript/concepts/dates/terms/getTime/getTime.md

```diff
@@ -0,0 +1,28 @@
+---
+Title: '.getTime()' 
+Description: 'Returns the number of milliseconds since 1st January 1970' 
+Subjects: 
+  - 'Web Development'
+Tags: 
+  - 'Date'
+  - 'Methods'
+CatalogContent: 
+  - 'path/web-development'
```

Comment:
```suggestion
  - 'path/web-development'
  - 'introduction-to-javascript'
```
Generally we have two entries for CatalogContent, one free, one paid.

Reply:
Ah ok noted thanks I wasn't sure about that will update accordingly